opendebates/tests/test_vote.py

```diff
@@ -294,6 +306,53 @@ def test_vote_user_disabled_captcha(self):
         refetched_sub = Submission.objects.get(pk=self.submission.pk)
         # ... and in DB
         self.assertEqual(self.votes + 1, refetched_sub.votes)
+        self.assertEqual(self.current_votes + 1, refetched_sub.current_votes)
+
+    def test_vote_after_previous_debate(self):
+        "Votes after the previous debate get tracked separately."
+        mode, _ = SiteMode.objects.get_or_create()
+        mode.previous_debate_time = timezone.make_aware(
+            datetime.datetime(2016, 1, 1, 12)
+        )
+        mode.save()
+
+        data = {
+            'email': self.voter.email,
+            'zipcode': self.voter.zip,
+            'g-recaptcha-response': 'PASSED'
+        }
+        rsp = self.client.post(self.submission_url, data=data,
+                               HTTP_X_REQUESTED_WITH='XMLHttpRequest')
+        self.assertEqual(200, rsp.status_code)
+        json_rsp = json.loads(rsp.content)
+        # vote is incremented in JSON response
+        self.assertEqual(self.votes + 1, json_rsp['tally'])
+        refetched_sub = Submission.objects.get(pk=self.submission.pk)
+        # ... and in DB
+        self.assertEqual(self.votes + 1, refetched_sub.votes)
+        self.assertEqual(self.current_votes + 1, refetched_sub.current_votes)
+
+    def test_vote_before_previous_debate(self):
+        "Votes before the previous debate don't get tracked yet."
+        mode, _ = SiteMode.objects.get_or_create()
+        mode.previous_debate_time = timezone.now() + datetime.timedelta(days=1)
+        mode.save()
+
+        data = {
+            'email': self.voter.email,
+            'zipcode': self.voter.zip,
+            'g-recaptcha-response': 'PASSED'
+        }
+        rsp = self.client.post(self.submission_url, data=data,
+                               HTTP_X_REQUESTED_WITH='XMLHttpRequest')
+        self.assertEqual(200, rsp.status_code)
+        json_rsp = json.loads(rsp.content)
+        # vote is incremented in JSON response
+        self.assertEqual(self.votes + 1, json_rsp['tally'])
+        refetched_sub = Submission.objects.get(pk=self.submission.pk)
+        # ... and in DB
+        self.assertEqual(self.votes + 1, refetched_sub.votes)
+        self.assertEqual(self.current_votes, refetched_sub.current_votes)
```

Comment:
It took me a moment to find the actual thing being tested in these tests -- perhaps a comment above this line (and the corresponding line in the previous test) would be in order?

Optionally, could any of the other boilerplate be removed, since it appears to be well-tested elsewhere?